talksapp/main.go

```diff
@@ -35,7 +35,6 @@ var (
 	}
 	homeArticle  = loadHomeArticle()
 	contactEmail = "golang-dev@googlegroups.com"
-	github       = httputil.NewAuthTransportFromEnvironment(nil)
```

Comment:
Why don't you move this line to just before line 124?
Seems a shame not to reuse the code in httputil.